WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -94,8 +94,13 @@ extension XCTestCase {
     }
 
     public func waitAndTap( _ element: XCUIElement) {
-        if element.waitForExistence(timeout: 5) {
-            element.tap()
+        var retries = 0
+        let maxRetries = 10
+        if element.waitForIsHittable(timeout: 10) {
+            while element.exists && retries < maxRetries {
```

Comment:
This is my only note for this PR: wdyt of using

```suggestion
            while element.isHittable && retries < maxRetries {
```

here? Firstly, this is what the check one line above is about. Secondly, if an element is hittable, it exists, but not the other way around, as the [doc says](https://developer.apple.com/documentation/xctest/xcuielement/1500760-exists). It's always possible to tap a hittable element, but it's not so for the existing one. So, by going for `isHittable` we reduce (by a miniscule amount, probably) a chance of error.

Totally optional, though 🙂

Reply:
Oh, yeah! It makes total sense. Thanks, @pachlava!

I've changed it in [f7cb326](https://github.com/wordpress-mobile/WordPress-iOS/pull/19095/commits/f7cb32646ec6e64185523c46b960478ef3a92bf5).

Reply:
Thanks for addressing this, @tiagomar! 🚀